certified-connectors/SIGNL4 - Mobile Alerting/apiDefinition.swagger.json

```diff
@@ -0,0 +1,15101 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "contact": {
+      "name": "Derdack GmbH",
+      "url": "https://www.signl4.com/feedback-we-love-hear-from-you/",
+      "email": "success@signl4.com"
+    },
+    "description": "SIGNL4 offers critical mobile alerting, incident response and service dispatching for operating critical infrastructure. It alerts you persistently via app push, SMS text and voice calls including tracking, escalation, collaboration and duty planning.",
+    "license": {
```

Comment:
@rons4 the license section can be removed.

Reply:
Done.